crypto/rand/rand_unix.c

```diff
@@ -238,10 +246,20 @@ static size_t sysctl_random(char *buf, size_t buflen)
 
 /*
  * syscall_random(): Try to get random data using a system call
- * returns the number of bytes returned in buf, or <= 0 on error.
+ * returns the number of bytes returned in buf, or < 0 on error.
  */
-int syscall_random(void *buf, size_t buflen)
+static ssize_t syscall_random(void *buf, size_t buflen)
 {
+    /*
+     * Note: 'buflen' equals the size of the buffer which is used by the
+     * get_entropy() callback of the RAND_DRBG. It is roughly bounded by
+     *
+     *   2 * DRBG_MINMAX_FACTOR * (RAND_DRBG_STRENGTH : 8) = 2^13
```

Comment:
`: 8` should that be `/ 8` to keep C syntax?

Reply:
I'd prefer this changed but it isn't essential.

Reply:
Ups, you're right! I'll change it. Seems like I had too many math sessions with my daughter recently ;-).

Reply:
I know the feeling :)